README.md

```diff
@@ -87,7 +87,7 @@ $ pyenv which curl
 
 ## License
 
-(The MIT License)
+(The MIT License)](LICENSE)
```

Comment:
I think lparen '(' at the beginning should be replaced by lbracket '[' 🤔

Reply:
Plus, I guess rparen after "License" could be unnecessary.

Reply:
My bad, I think my editor erased a bracket.